WooCommerce/Classes/Authentication/Prologue/LoginPrologueViewController.swift

```diff
@@ -152,28 +152,19 @@ private extension LoginPrologueViewController {
     /// Returns the Disclaimer Attributed Text (which contains a link to the Jetpack Setup URL).
     ///
     var disclaimerAttributedText: NSAttributedString {
-        let disclaimerText = NSLocalizedString("This app requires Jetpack to connect to your Store. Read the ", comment: "Login Disclaimer Text and Jetpack config instructions")
+        let disclaimerText = NSLocalizedString("This app requires Jetpack to connect to your Store. <br /> Read the <a href=\"https://jetpack.com/support/getting-started-with-jetpack/\">configuration instructions</a>.", comment: "Login Disclaimer Text and Jetpack config instructions")
         let disclaimerAttributes: [NSAttributedString.Key: Any] = [
             .font: UIFont.font(forStyle: .caption1, weight: .thin),
             .foregroundColor: UIColor.white
         ]
 
-        let readText = NSLocalizedString("configuration instructions", comment: "Login Disclaimer Linked Text")
-        let readAttributes: [NSAttributedString.Key: Any] = [
-            .link: URL(string: WooConstants.jetpackSetupUrl) as Any,
-            .font: UIFont.font(forStyle: .caption1, weight: .thin)
-        ]
-
-        let readAttrText = NSMutableAttributedString(string: readText, attributes: readAttributes)
-
-        let endSentenceText = "."
-        let endSentenceAttrText = NSMutableAttributedString(string: endSentenceText, attributes: disclaimerAttributes)
+        guard let disclaimerAttrText = disclaimerText.htmlToAttributedString else {
+            return NSAttributedString()
+        }
 
-        let disclaimerAttrText = NSMutableAttributedString(string: disclaimerText, attributes: disclaimerAttributes)
-        disclaimerAttrText.append(readAttrText)
-        disclaimerAttrText.append(endSentenceAttrText)
+        disclaimerAttrText.addAttributes(disclaimerAttributes, range: NSRange(location: 0, length: disclaimerAttrText.length))
 
-        return disclaimerAttrText
+        return disclaimerAttrText as NSAttributedString
```

Comment:
And I believe you can drop off the cast as the return type on the computed property should do the casting automagically. 😄